integration_tests/tests/cache/basic.tests.js

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+"use strict";
+
+const chai = require("chai");
+const nock = require("nock");
+const { StaticClient, BaseClient }= require("@commerce-apps/core");
```

Comment:
Missing a space here (`}=`) and in a few other files.

Reply:
In process to add eslint, missed it in the commits..